### ISSUE TITLE
TWITCH Instance 14 died: Error:Page.goto: net::ERR_TUNNE... Please see cvamp.log.

### ISSUE THREAD

**Issue by natan522** [KevinBytesTheDust/CVAmp#311]
Hello, I'm having a problem running the program and this error appears.

![Captura de tela 2024-10-19 195249](https://github.com/user-attachments/assets/dfb30611-75d4-49f9-b908-a6c1cafa2270)

**Comment by jnwright** [comment]
also having this error. most instances immediately die, and then when they do work, they die a few minutes after.

**Comment by KevinBytesTheDust** [comment]
Tunnel error is related to the proxy. Can you make sure that you use the active proxies, that have sufficient traffic on them?
The free proxies from [webshare](https://github.com/KevinBytesTheDust/cvamp/wiki/Webshare.io-Proxies-Guide) have limited traffic.

**Comment by ThreePerCento** [comment]
What if I want to use a proxy not from Webshare, I only need to specify the IP and port? Do I need to specify the login and password and where do I get them from?

**Comment by 1234Testaccount1234** [comment]
I'm running into this same issue and with Webshare using Static Residential Proxies with High Priority and High Concurrency. They just die  instantly.

**Comment by Cloudstrifeff7** [comment]
I have this problem too...

**Comment by Migren2009** [comment]
webshare asks for verifcations or they will close the account any help ??

**Comment by KevinBytesTheDust** [comment]
@Migren2009 Yes, [webshare ](https://blueloperlabs.ch/proxy/wp) does require verification after an unknown amount of consumption.
After verification, everything should work fine again.

@Darkestangel69 I am on the lookout for new proxy providers and have tested [proxyscrape](https://blueloperlabs.ch/proxy-ps/tf).
You are welcome to suggest additional providers for me to try and evaluate.
